loop_until_repro.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
This script needs a new home, maybe under cppsuite/tools or something? I don't know...

Reply:
[14a49a8](https://github.com/wiredtiger/wiredtiger/pull/11272/commits/14a49a8b0f9708fd399cfd5c13bed82e150e58df)

Reply:
👍 I've renamed to `test_live_restore_loop.sh` in [661d0e1](https://github.com/wiredtiger/wiredtiger/pull/11272/commits/661d0e1dcdf207dcbdb7baedfc39e59438f9a696)